operator/pkg/common/extensions/reconcile_scanner_db_password.go

```diff
@@ -34,7 +34,7 @@ func ReconcileScannerDBPassword(ctx context.Context, obj ScannerBearingCustomRes
 
 func reconcileScannerDBPassword(ctx context.Context, obj ScannerBearingCustomResource, client ctrlClient.Client, config reconcileScannerDBPasswordConfig) error {
 	run := &reconcileScannerDBPasswordExtensionRun{
-		SecretReconciliator:  NewSecretReconciliator(client, obj),
+		SecretReconciliator:  NewSecretReconciliator(client, obj, OwnershipStrategyOwnerReference),
```

Comment:
Shouldn't this be using the other strategy? I guess this component has the same exact issue as central DB? 🤔 
If not, please add a comment explaining why this one is different.

Reply:
Added a comment to explain why the OwnerReference strategy is used